src/sql-parser/tests/testdata/ddl

```diff
@@ -1401,13 +1401,12 @@ ALTER SECRET secret AS decode('new c2VjcmV0Cg==', 'base64')
 =>
 AlterSecret(AlterSecretStatement { name: Name(UnresolvedObjectName([Ident("secret")])), if_exists: false, value: Function(Function { name: UnresolvedObjectName([Ident("decode")]), args: Args { args: [Value(String("new c2VjcmV0Cg==")), Value(String("base64"))], order_by: [] }, filter: None, over: None, distinct: false }) })
 
-
 parse-statement
-CREATE CONNECTOR conn1 FOR KAFKA BROKER 'kafka:1234' WITH (security_protocol = 'SASL_SSL', sasl_mechanisms = 'PLAIN')
+CREATE CONNECTOR newconn FOR KAFKA BROKER 'kafka:1234' SECURITY SASL MECHANISM PLAIN USERNAME 'phineas' PASSWORD 'butterfat'
```

Comment:
if the variables must be in exactly this order, I'd like to see some tests demonstrating some out of order behavior